### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rhcarvalho**
@mfojtik asked, so here we go!

[typokiller](https://github.com/rhcarvalho/typokiller) found very few typos here, awesome!

### CODE REVIEW

**Comment by mfojtik** [comment]
awesome! thanks!

[merge]

File: pkg/build/strategies/sti/sti_test.go [hunk 1]
```diff
@@ -149,17 +149,17 @@ func TestBuild(t *testing.T) {
 			t.Errorf("Unexpected optional scripts requested: %#v", fh.SetupOptional)
 		}
 
-		// Verify that determineIncremental was called
+		// Verify that Determine was called
```
Comment (soltysh) [inline pkg/build/strategies/sti/sti_test.go]: This is bad change, I prefer the stay with the old one.

File: pkg/build/strategies/sti/sti_test.go [hunk 2]
```diff
@@ -149,17 +149,17 @@ func TestBuild(t *testing.T) {
...
 		if !fh.DetermineIncrementalCalled {
 			t.Errorf("Determine incremental was not called.")
 		}
 
-		// Verify that saveartifacts was called for an incremental build
+		// Verify that Save was called for an incremental build
```
Comment (soltysh) [inline pkg/build/strategies/sti/sti_test.go]: Same here.

Comment (rhcarvalho) [inline pkg/build/strategies/sti/sti_test.go]: I based the changes in the fact that the interface for [`IncrementalBuilder`](https://github.com/openshift/source-to-image/blob/d2c8661cf024ad774210a218b03717b2673d53e1/pkg/build/interfaces.go#L29-L32) is:
```go
type IncrementalBuilder interface {
	Determine(*api.Request) error
	Save(*api.Request) error
}
```

File: pkg/build/strategies/sti/sti_test.go (hunk 1 above)
Comment (mfojtik) [inline pkg/build/strategies/sti/sti_test.go]: determineIncremental is no more

Comment (soltysh) [inline pkg/build/strategies/sti/sti_test.go]: Yeah, both of you're right, I can't get used to your refactor :wink:

**Pull request merged**